test/e2e/marketplace_test.go

```diff
@@ -123,9 +119,6 @@ func defaultCreateTest(t *testing.T, f *test.Framework, ctx *test.TestCtx) error
 			Timeout:       cleanupTimeout,
 			RetryInterval: cleanupRetryInterval,
 		})
```

Comment:
why are we removing this error check?

Reply:
This should not have been removed - my mistake.